DigitalOffice/Services/FileService/Mappers/FileMapper.cs

```diff
@@ -0,0 +1,33 @@
+﻿using FileService.Database.Entities;
+using FileService.Mappers.Interfaces;
+using FileService.Models;
+using System;
+
+namespace FileService.Mappers
+{
+    public class FileMapper : IMapper<DbFile, FileRequest>, IMapper<FileRequest, DbFile>
+    {
+        public FileRequest Map(DbFile dbFile)
```

Comment:
For what purpose we want map DbFile to FileRequest?

Reply:
Well, for the time being we do not have FileResponse model so I treat FileRequest as the only frontend-model.
Also, it is needed when you want to get an actual file. How else are we gonna get it if not map it from DbFile to just File?

Remember, Spartak taught as that we always have different models and transfer data from one thing to another.

Reply:
I think it is wrong to consider FileRequest as the only frontend-model. The model for the response may differ from the model for the request. It would be more correct to create a separate model for the responce, but then this method will be useless.

Reply:
I agree that it is wrong. I mean that while we do not have Responses, I decided to treat this model as the only one.

Seems I better have created a Response instead

Reply:
Fixed it, creating a separate model called "File".

Changed mapping from DbFile -> FileRequest to
DbFile -> File